### DISCUSSION TITLE
Depois que terminei um projeto freelancer em front-end/desenvolvimento web, como entrego esse projeto para o cliente?

### DISCUSSION THREAD

**Discussion by ThiagoCarrijo1** [frontendbr/forum#2378] [Discussão]
Estou iniciando, já estudei HTML e CSS e estou perto de iniciar alguns projetos e tenho essa duvida, digamos que terminei o freela de front-end mas e agora o que eu faço, como entrego para o cliente, o que eu preciso fazer? Enviar para ele a pasta do projeto? procurei em tudo que é lugar e não tem ninguém ensinando o que faz no final do freela pra entregar o projeto concluído para o cliente, é uma pergunta meio boba mas sou iniciante e não tenho ideia de como acontece isso.

**Comment by felquis** [comment]
Geralmente compartilho acesso ao repositório, servidores, etc...
e continuo atrelado ao projeto indeterminadamente.

Você é enternamente responsável pelos projetos que cria aos outros, a não ser que você passe para outro dev no futuro e lave as mãos.

caso seja um freela que não é você quem administra os servidores, dns, etc... nestes casos eu só finalizo meus trabalho em um pull request, ele faz merge e depois de concluido eles me removem do repositório e nunca mais ouço falar.

Espero ter ajudado

**Comment by RyanAraujoo** [comment]
Se o seu produto já está pronto, é necessário inicializar os testes.

`Meu produto está acessível?`
`Meu produto vai ser usado pelo usuário final?`
`Meu produto está com um design agradável?`
`Como posso garantir as 5 estrelas pelo meu trabalho?`

Finalizou? 
Dê o acesso ao repositório do projeto c/ documentação de uso.

Ele já quer o produto no ar?
Ah, dependendo do projeto, pode ser hospedado até pelo próprio github pages e só compra o dominio.
